### ISSUE TITLE
Step is failing when there is no dsym in version 3.1.0

### ISSUE THREAD

**Issue by omarzl** [bitrise-steplib/steps-xcode-archive#210]
We have an app that is not generating dsym files in specific configurations, we want it like that.

Using version 3.0.0 of this step it shows the message: "no app dsyms found" and the step finishes successfully.

<img width="1566" alt="image" src="https://user-images.githubusercontent.com/6267487/102408526-51dbfe80-3fb3-11eb-8fa6-89fae13cad18.png">

But now with version 3.1.0 the step is failing

<img width="1638" alt="image" src="https://user-images.githubusercontent.com/6267487/102408590-746e1780-3fb3-11eb-96d7-83034a59c421.png">

I trigger two builds setting the option "Export all dsyms" with NO and YES but in both cases it is failing.

**Comment by bitce** [comment]
Hi @omarzl, Can you please include a build URL here for both instances? Thanks!

**Comment by omarzl** [comment]
Hello!
Here is a successful build using version 3.0.0
https://app.bitrise.io/build/fe1c48b45ec8f58f#?tab=log

A failing build using version 3.1.0 with export_all_dsyms: 'yes'
https://app.bitrise.io/build/c0b077c0653b3714#?tab=log

And a failing build using version 3.1.0 with export_all_dsyms: 'no'
https://app.bitrise.io/build/4ef01a9c24ffa090#?tab=log

For that specific Xcode configuration we are not generating dsym:
`DEBUG_INFORMATION_FORMAT = dwarf;`

**Comment by patrickbussmann** [comment]
I have the same issue.
Updated from 2.x.x to 3.x.x and now I'm getting:

```
▸ Export Succeeded
Exporting outputs...
The xcarchive path is now available in the Environment Variable: BITRISE_XCARCHIVE_PATH (value: /var/folders/6q/wgy6jtp12w5gzgm9lzcglpqw0000gn/T/__archive__032907782/App.xcarchive)
The xcarchive zip path is now available in the Environment Variable: BITRISE_XCARCHIVE_ZIP_PATH (value: /Users/vagrant/deploy/App.xcarchive.zip)
The app directory is now available in the Environment Variable: BITRISE_APP_DIR_PATH (value: /Users/vagrant/deploy/App.app)
The ipa path is now available in the Environment Variable: BITRISE_IPA_PATH (value: /Users/vagrant/deploy/App.ipa)
Failed to export dsyms, error: no dsym found
```

https://app.bitrise.io/build/5995764b9a311f37#?tab=log

Tried with enabled and disabled "export dsyms" option. Both failed.

Downgrade to 2.x.x works again:
https://app.bitrise.io/build/0550c45f3792fc99#?tab=log

There is the log:
```
▸ Export Succeeded
Exporting outputs...
The xcarchive path is now available in the Environment Variable: BITRISE_XCARCHIVE_PATH (value: /var/folders/6q/wgy6jtp12w5gzgm9lzcglpqw0000gn/T/__archive__803678785/App.xcarchive)
The xcarchive zip path is now available in the Environment Variable: BITRISE_XCARCHIVE_ZIP_PATH (value: /Users/vagrant/deploy/App.xcarchive.zip)
The app directory is now available in the Environment Variable: BITRISE_APP_DIR_PATH (value: /Users/vagrant/deploy/App.app)
The ipa path is now available in the Environment Variable: BITRISE_IPA_PATH (value: /Users/vagrant/deploy/App.ipa)
no app dsyms found
The dSYM dir path is now available in the Environment Variable: BITRISE_DSYM_DIR_PATH (value: /var/folders/6q/wgy6jtp12w5gzgm9lzcglpqw0000gn/T/__dsyms__007416069)
The dSYM zip path is now available in the Environment Variable: BITRISE_DSYM_PATH (value: /Users/vagrant/deploy/App.dSYM.zip)
```

Interesting!

**Comment by bitce** [comment]
Hi @patrickbussmann and @omarzl! Thanks for the details and your patience. The team is taking a look, the fix should be simple, will let you know here!

**Comment by bitce** [comment]
Version 3.1.1 of the Xcode Archive step was just realized, and it should fix this problem, let us know what you experience!

**Comment by mateherber** [comment]
Hi @omarzl,

We've fixed this issue supposedly in `3.1.1` version of the step via this [PR](https://github.com/bitrise-steplib/steps-xcode-archive/pull/209). I'm closing this issue now. Should you experience any issues with the step please open a new issue with our new [bug templates](https://github.com/bitrise-steplib/steps-xcode-archive/issues/new/choose)!
